common/src/test/java/org/hiero/block/common/utils/PreconditionsTest.java

```diff
@@ -459,6 +480,189 @@ void testRequirePositivePowerOf10Fail(final int toTest) {
                 .withMessage(expectedTestMessage);
     }
 
+    /**
+     * This test aims to verify that the
+     * {@link Preconditions#requireRegularFile(Path)} will return the input
+     * 'toTest' parameter if the regular file check passes.
+     * Test includes overloads.
+     */
+    @Test
+    void testSuccessfulRequireRegularFile() throws IOException {
```

Comment:
It would be good for test stuff to move out of common main module. We can add a common test fixtures module if needed across apps, otherwise add to app fixtures module.